docs-conceptual/azps-2.8.0/install-az-ps-msi.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Install Azure PowerShell with an MSI
+description: How to install Azure PowerShell without PowerShellGet using an MSI
+author: sptramer
+ms.author: sttramer
+manager: carmonm
+ms.devlang: powershell
+ms.topic: conceptual
+ms.date: 10/22/2019
+---
+
+# Install Azure PowerShell on Windows with MSI
+
+This article explains how to install Azure PowerShell on Windows using an MSI installer. The MSI
+installer is provided for environments where the PowerShell Gallery may be blocked by a firewall, or
+an offline installer is needed. The recommended way to install Azure PowerShell
+is with PowerShellGet. For instructions on using PowerShellGet to install Azure PowerShell,
+see [Install Azure PowerShell with PowerShellGet](install-az-ps.md).
+
+## Requirements
+
+The MSI installer for Azure PowerShell works __only__ for PowerShell 5.1 on Windows. For installation on
+non-Windows platforms or later versions of powershell, [Install with PowerShellGet](install-az-ps.md).
+To check your PowerShell version, run the command:
+
+```powershell-interactive
+$PSVersionTable.PSVersion
+```
+
+To use Azure PowerShell in PowerShell 5.1, you need to:
+
+1. Update to [Windows PowerShell 5.1](/powershell/scripting/install/installing-windows-powershell#upgrading-existing-windows-powershell) if needed. If you're on Windows 10, you already
+  have PowerShell 5.1 installed.
+2. Install [.NET Framework 4.7.2 or later](/dotnet/framework/install).
+
+## Install or update on Windows using the MSI Package
+
+Azure PowerShell for Windows is installed using the MSI file available from
+[GitHub](https://github.com/Azure/azure-powershell/releases/tag/v2.8.0-October2019). If you have installed earlier
+versions of Azure modules as an MSI, the installer automatically removes them. The MSI package installs
+modules in `${env:ProgramFiles}\WindowsPowerShell\Modules`.
+
+To start working with Azure PowerShell, sign in with your Azure credentials.
+
+```powershell-interactive
+# Connect to Azure with an interactive dialog for sign-in
+Connect-AzAccount
+```
+
+> [!NOTE]
+>
+> If you've disabled module autoloading, you need to manually import the module with `Import-Module Az`. Because of
+> the way the module is structured, this can take a few seconds.
```

Comment:
```suggestion
> the way the module is structured, this can take up to a minute.
```

Only reason for change is that it can take upto a minute to import the Az Module as the below screenshot shows
![image](https://user-images.githubusercontent.com/6355225/67320194-c3cacc00-f505-11e9-99d3-ee9e1d24bfc9.png)